data_collection/gazette/spiders/base.py

```diff
@@ -188,3 +188,84 @@ def get_gazette(self, document):
             file_urls=(document[0],),
             power="executive",
         )
+
+
+class DoemGazetteSpider(BaseGazetteSpider):
+    """
+    Base spider for all cities listed on https://doem.org.br
+    """
+
+    allowed_domains = ["doem.org.br"]
+    start_date = date(2009, 1, 1)
```

Comment:
Is this valid for all cities?

Reply:
Yes, as far as I could see they're all after 2013.